services/workflows-service/src/common/filters/prisma-client-validation-filter/utils/PrismaValidationExceptionParser.ts

```diff
@@ -0,0 +1,37 @@
+import { PrismaValidationExceptionParseResult } from './types';
+import { Prisma } from '@prisma/client';
+
+/**
+ * PrismaValidationExceptionParser
+ * @constructor
+ * @param {Prisma.PrismaClientValidationError} exception - Prisma validation exception
+ * @description Parsing exception for errors and convert them to JSON format
+ * Excluding referernces to prisma and application internals
+ * @method parse @returns {PrismaValidationExceptionParseResult} record of entries where key is paramName and its value errorReason
+ */
+
+export class PrismaValidationExceptionParser {
+  // Looking for matches in following string format: Argument {{paramName}}: Got {{errorReason}} on prisma
+  private pattern = new RegExp(/Argument ([^:]+):[^]+?Got ([^]+?) on prisma/, 'g');
```

Comment:
I checked a common issue - an unknown argument and got empty response:
```
{
    "statusCode": 400,
    "message": "Validation failed.",
    "result": {}
}
```

the error string is: 

```
Unknown arg `companyName1` in data.companyName1 for type BusinessCreateInput. Did you mean `companyName`? Available args:
```

Reply:
If you need the curl i made, ping me on slack

Reply:
@alonp99 Added handler for this kind of errors